plugins/woocommerce/tests/e2e-pw/tests/merchant/create-variable-product.spec.js

```diff
@@ -14,7 +14,7 @@ const defaultAttributes = [ 'val2', 'val1', 'val2' ];
 const stockAmount = '100';
 const lowStockAmount = '10';
 
-test.describe( 'Add New Variable Product Page', () => {
+test.describe.serial( 'Add New Variable Product Page', () => {
```

Comment:
I had to use `serial` here so that if a middle test fails, earlier tests will be retried even if they already passed.
Using serial is discouraged in the Playwright docs, but it seems to be a quick solution in tests that depend on earlier tests.